src/goto-instrument/contracts/instrument_spec_assigns.cpp

```diff
@@ -118,6 +175,26 @@ void instrument_spec_assignst::track_static_locals(goto_programt &dest)
   }
 }
 
+void instrument_spec_assignst::track_static_locals(goto_programt &dest)
+{
+  collect_static_locals_from_root_function(function_id, dest);
+}
+
+void instrument_spec_assignst::track_static_locals_from_calls(
```

Comment:
Why don't we cover this functions? (see Codecov reports).

Reply:
removed